src/ADAL.Common/Constants.cs

```diff
@@ -331,6 +336,7 @@ internal static partial class AdalErrorMessage
         public const string SpecifyAnyUser = "If you do not need access token for any specific user, pass userId=UserIdentifier.AnyUser instead of userId=null.";
         public const string IntegratedAuthFailed = "Integrated authentication failed. You may try an alternative authentication method";
         public const string DuplicateQueryParameterTemplate = "Duplicate query parameter '{0}' in extraQueryParameters";
+        public const string InteractionRequired = "interaction_required";
```

Comment:
Should this not be a more verbose error message (not just the same as the error_code)?